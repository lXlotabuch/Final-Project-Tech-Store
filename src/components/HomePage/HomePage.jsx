import React, { useEffect } from 'react';
import {BannerSlider} from '../BannerSlider/BannerSlider';
import {HomepageFollowUs} from '../BrandsAndFollow/FollowUsInstagram';
import {HomepageBrands} from '../BrandsAndFollow/HomePageBrands';
import Description from '../Description/Description';
import NewProductsSlider from '../NewProductsSlider/NewProductsSlider';
import ReviewSlider from '../ReviewSlider/ReviewSlider';
import {Catalog as ProductsPreview } from '../ProductsPreview/Catalog';

const HomePage = () => {
  useEffect(() => {
    window.scrollTo(0, 0)
  }, [])

  return (
    <div>
      <BannerSlider />
      <NewProductsSlider />
      <ProductsPreview />
      <Description />
      <HomepageBrands />
      <HomepageFollowUs />
      <ReviewSlider />
    </div>
  )
}

export default HomePage;