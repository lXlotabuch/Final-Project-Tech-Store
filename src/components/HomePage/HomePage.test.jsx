import {render} from '@testing-library/react'
import React from 'react'
import { HashRouter as Router } from 'react-router-dom'
import { Provider } from 'react-redux'
import { store } from '../../store/index'
import HomePage from './HomePage'
import { machMedia } from '../../mocks/matchMedia.mock'

describe('HomePage', () => {
  beforeEach(() => machMedia())

  it('render', () => {
    const {asFragment} = render(
      <Provider store={store}>
        <Router>
          <HomePage />
        </Router>
      </Provider>
    )

    expect(asFragment).toMatchSnapshot()
  })
});
