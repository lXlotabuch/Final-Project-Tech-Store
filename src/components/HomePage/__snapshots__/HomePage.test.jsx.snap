// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage render 1`] = `[Function]`;
